openlr/openlr_match_quality/openlr_assessment_tool/map_widget.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "qt/qt_common/map_widget.hpp"
+
+namespace
+{
+class PointsController;
+}  // namespace
+
+class Framework;
+
+class MapWidget : public qt::common::MapWidget
+{
+  Q_OBJECT
+
+  using Base = qt::common::MapWidget;
+
+  enum class Mode
+  {
+    Normal,
+    TrafficMarkup
+  };
+
+public:
+  MapWidget(Framework & framework, bool apiOpenGLES3, QWidget * parent);
+  ~MapWidget() override;
```

Comment:
Yes, I can. Should I override it than? Would it be better to remove it at all since we have one in base class?